rewards/explorer.py

```diff
@@ -36,10 +36,9 @@ def convert_from_eth(block) -> Dict[str, int]:
     """
     timestamp = env_config.get_web3().eth.getBlock(block)["timestamp"]
     return {
-        "eth": block,
-        # "bsc": get_block_by_timestamp("bsc", timestamp) - 1000,
-        "polygon": get_block_by_timestamp("polygon", timestamp) - 1000,
-        "arbitrum": get_block_by_timestamp("arbitrum", timestamp) - 1500,
+        Network.Ethereum: block,
+        Network.Polygon: get_block_by_timestamp(Network.Polygon, timestamp) - 1000,
```

Comment:
magic number again here and below

Reply:
We can remove this function - its not needed with single chain boost